R/data.table.R

```diff
@@ -823,6 +836,7 @@ chmatch2 <- function(x, table, nomatch=NA_integer_) {
       } else {
         stop("When with=FALSE, j-argument should be of type logical/character/integer indicating the columns to select.") # fix for #1440.
       }
+>>>>>>> 7a662249f1d7755179ce7ec2cddc5c3afdb6c951
```

Comment:
standard procedure after each merge conflict, ctrl+f: `>>>`, `<<<`, `===`

Reply:
yes, noticed this, but there was a conflicting logic change, need to spend a few minutes to make sure the merge is correct... thanks for the eyes already :)